web/themes/custom/westy/src/components/icon/readme.md

```diff
@@ -0,0 +1,17 @@
+# Icon
+
+## Props
+
+- `name`: string - Name of icon, must be one from the "All Icons" list.
+    - Is the filename of an svg in the `web/themes/custom/westy/src/icon/svgs/` folder
```

Comment:
This is not at all accurate anymore. We are just using FA for the icons. Please remove this readme or make it accurate.